src/libs/SDLWindowManager/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+project(SDLWindowManager)
+
+find_package(SDL2 REQUIRED)
+
+add_library(${PROJECT_NAME} STATIC
+    SDLWindowManager.hpp
```

Comment:
in the new code we put public headers in the `include` subfolder. See https://github.com/storm-devs/storm-engine/tree/develop/src/libs/SteamApi